docs/source/technical_tutorials/visualization/interval_color_mapping.ipynb

```diff
@@ -2,19 +2,24 @@
  "cells": [
```

Comment:
<p>Did we need to change the collection/item on this? I get CRS invalid when trying to pull up the cog/statistics on this item, but the AfriSAR_AGB_Maps_1681___1/Rabi_AGB_50m.tif seems to be working just fine - https://titiler-stac.maap-project.org/cog/statistics?url=s3://nasa-maap-data-store/file-staging/nasa-map/AfriSAR_AGB_Maps_1681___1/Rabi_AGB_50m.tif</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/253/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>